test/test_suite.bats

```diff
@@ -249,14 +249,14 @@
 }
 
 @test "Fail on invalid argument" {
-  run bash -c './pihole-FTL abc'
+  run bash -c '/home/pihole/pihole-FTL abc'
```

Comment:
Is there any way to not hard-code the binary path and name? This is not good for local testing.

Reply:
I don't know how to transport a variable into the `bats` tests. If you know how to do it, you are more than invited to add this. We could also `cd` into this directory and just run `./pihole-FTL` if that is more convenient fir local tests.

Reply:
You can probably use an environment variable, but if `cd`-ing is easier (and doesn't introduce more issues with local testing) then that's fine.

Reply:
For now, we can leave this as is and execute the tests as part of the CI pipeline locally (`circleci local execute --job x86_64`).